echopype/echodata/combine.py

```diff
@@ -90,10 +77,12 @@ def combine_echodata(echodatas: List[EchoData], combine_attrs="override") -> Ech
     -----
     UserWarning
         If the `sonar_model` of the input `EchoData` objects is `"EK60"` and any `EchoData` objects
-        have non-monotonically increasing `ping_time`s or `location_time`s,
-        the `ping_time`s or `location_time`sin the output `EchoData` object will be increased
+        have non-monotonically increasing `ping_time` or `location_time` values,
+        the `ping_time` or `location_time` values in the output `EchoData` object will be increased
         starting at the timestamp where the reversal occurs such that all
-        `ping_time`s and `location_time`s in the output are monotonically increasing.
+        `ping_time` and `location_time` values in the output are monotonically increasing.
+        Additionally, the original `ping_time` and `location_time` values will be stored
+        in the provenance group, although this behavior may change in future versions.
```

Comment:
@emiliom : there's also `mru_time` that was added to this check later on, which didn't make its way to this docstring. Could you add it in?

@imranmaj : wanted to double check with you to make sure that the `ping_time_*` coordinates in AD2CP files are not included in this checking -- from the code it does not seem like they are.

Thanks both!

Reply:
Sure, I can add mru_time in a bit.